src/types/network-config.ts

```diff
@@ -6,14 +6,10 @@ export type NetworkConfig = {
     guardian: AddressConfig;
     management: AddressConfig;
     treasury: AddressConfig;
-    managementFee: number;
```

Comment:
This file defines a NetworkConfig used by each network. I've removed the fixed management & performance fees as Robo vaults can have different fees. We just use whatever fees are returned by RPC calls.